lightning/pkg/importer/precheck_impl.go

```diff
@@ -576,32 +576,43 @@ func (ci *localTempKVDirCheckItem) Check(ctx context.Context) (*precheck.CheckRe
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	localAvailable := int64(storageSize.Available)
+	localAvailable := int64(storageSize.Available)           //Available disk space in bytes
+	availableStr := units.BytesSize(float64(localAvailable)) //Human-readable availabe space
+
 	estimatedDataSizeResult, err := ci.preInfoGetter.EstimateSourceDataSize(ctx)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 	estimatedDataSizeWithIndex := estimatedDataSizeResult.SizeWithIndex
+	estimatedStr := units.BytesSize(float64(estimatedDataSizeWithIndex)) //Human-readable estimated size
+
+	diskQuota := int64(ci.cfg.TikvImporter.DiskQuota)   //Configured disk quota in bytes
+	diskQuotaStr := units.BytesSize(float64(diskQuota)) //Human-readabledisk quota
+
+	//Warn the user if diskQuota is 0 or negative, as it's likely a misconfiguration
+	if diskQuota <= 0 {
+		log.FromContext(ctx).Warn("`tikv-importer.disk-quota` is set to 0 or less; please configure a valid positive value.")
```

Comment:
```suggestion
		log.FromContext(ctx).Warn("`tikv-importer.disk-quota` is set to 0 or less; please configure a valid positive value")
```
By usual, we don't add symbol at the end of the log